backend/ee/danswer/server/tenants/provisioning.py

```diff
@@ -61,3 +63,46 @@ def ensure_schema_exists(tenant_id: str) -> bool:
                 db_session.execute(stmt)
                 return True
             return False
+
+
+def user_owns_a_tenant(email: str) -> bool:
```

Comment:
this seems to check that the user is part of a tenant, does own imply being the first admin or something else?